anchor-reference-server/src/main/java/org/stellar/anchor/reference/AnchorEventServer.java

```diff
@@ -0,0 +1,107 @@
+package org.stellar.anchor.reference;
+
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.beans.factory.DisposableBean;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+import org.springframework.stereotype.Component;
+import org.stellar.anchor.event.models.AnchorEvent;
+import org.stellar.anchor.event.models.QuoteEvent;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Properties;
+
+@Component
+public class AnchorEventServer implements DisposableBean, Runnable {
+    private Thread thread;
+    private Consumer consumer;
+    private volatile boolean shutdown = false;
+
+    AnchorEventServer(){
+        this.thread = new Thread(this);
+        this.thread.start();
+    }
+
+    @Override
+    public void run(){
+
+        System.out.println("queue consumer server started ");
```

Comment:
I think you should have access to Log.info from this file. It would be better to log things using Log.info when we're in production.